tests/TestCase/Database/Type/DateTimeTypeTest.php

```diff
@@ -308,6 +308,42 @@ public function testMarshal($value, $expected)
         }
     }
 
+    /**
+     * Test that the marhsalled datetime instance always has the system's default timezone.
+     *
+     * @return void
+     */
+    public function testMarshalDateTimeInstance()
+    {
+        $expected = new Time('2020-05-01 23:28:00', 'Europe/Paris');
+
+        $result = $this->type->marshal($expected);
+        $this->assertEquals('UTC', $result->getTimezone()->getName());
+        $this->assertEquals($expected, $result->addHours(2));
```

Comment:
This is the same instance in `$expected` and `$result`, right? So, this is just comparison the same instance to itself.